instrumentation/otel-webserver-module/include/core/api/Payload.h

```diff
@@ -47,7 +47,6 @@ class RequestPayload
     std::string flavor;
     std::string client_ip;
     long port = 80;
-    long status_code = 200;
```

Comment:
No magic numbers here? We can find a better way right?

Reply:
This is a removed code.